config/coupons.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'enabled' => env('COUPONS_ENABLED', true),
```

Comment:
It feels a bit weird to create a separate file for this, I would add a `features` part to `app.php`